paragon/_overrides.scss

```diff
@@ -203,11 +203,62 @@ select.form-control {
 
 
 // Typography
+
 body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+h1, .h1 {
+  line-height: (44/16) * 1rem;
+  letter-spacing: -2%;
+
+  @media (max-width: map-get($grid-breakpoints, "sm")) {
+    line-height: (40/16) * 1rem;
+  }
+  .mobile-type & {
+    line-height: (40/16) * 1rem;
+  }
+}
+
+h2, .h2 {
+  line-height: (36/16) * 1rem;
+}
+
+h3, .h3 {
+  line-height: (28/16) * 1rem;
+}
+
+h4, .h4 {
+  line-height: (24/16) * 1rem;
+}
+
+h5, .h5 {
+  line-height: (20/16) * 1rem;
+}
+
+h6, .h6 {
+  line-height: (20/16) * 1rem;
+}
+
+.lead {
+  line-height: (36/16) * 1rem;
+}
+
+.small {
+  line-height: (24/16) * 1rem;
+}
+
+.x-small {
+  line-height: (20/16) * 1rem;
+}
+
+// todo add mobile
```

Comment:
What's keeping this from going in now?

Reply:
Ah good catch. There wasn't anything to add ultimately